terraform/stacks/main/stack.tf

```diff
@@ -3,7 +3,36 @@ terraform {
   }
 }
 
+
 provider "aws" {
+  # endpoints from # These endpoints are form https://docs.aws.amazon.com/govcloud-us/latest/UserGuide/using-govcloud-endpoints.html
+  # with help from https://github.com/Kaydub00/terraform-aws-fips/blob/master/govcloud-west.tf
+  endpoints {
+    # cloudtrail: Non-fips is n/a
+    # cloudwatch: non-fips is n/a
+    # cloudwatchevents: non-fips is n/a
+    # cloudwatchlogs: non-fips is n/a
+    # ec2             : non-fips is n/a
+    efs = "https://elasticfilesystem-fips.us-gov-west-1.amazonaws.com"
```

Comment:
Can we generalize these so they work with regions other than us-gov-west-1 and partitions other than us-gov?

Reply:
Not sure how. The EFS endpoints are:
* US E/W
  elasticfilesystem-fips.us-east-1.amazonaws.com
  elasticfilesystem-fips.us-east-2.amazonaws.com
  elasticfilesystem-fips.us-west-1.amazonaws.com
  elasticfilesystem-fips.us-west-2.amazonaws.com
* US Govcloud
  elasticfilesystem-fips.us-gov-east-1.amazonaws.com
  elasticfilesystem-fips.us-gov-west-1.amazonaws.com

According to https://registry.terraform.io/providers/hashicorp/aws/latest/docs/guides/custom-service-endpoints:
> If multiple, different Terraform AWS Provider configurations are required, see the [Terraform documentation on multiple provider instances](https://www.terraform.io/docs/configuration/providers.html#alias-multiple-provider-instances) for additional information about the alias provider configuration and its usage.

Reply:
I think we could parametrize this with, e.g. `http://elasticfilesystem-fips.${var.aws_region}.amazonaws.com`, but we'll have to do it for each provider (not a huge deal, just tedious and probably not super-obvious in some places)